Fix for CMake 4
https://bugs.gentoo.org/951630
https://github.com/pezmaster31/bamtools/commit/a610d2c0150d4b821aa4096e57eb77e4b0c14aa7
From a610d2c0150d4b821aa4096e57eb77e4b0c14aa7 Mon Sep 17 00:00:00 2001
From: David Seifert <SoapZA@users.noreply.github.com>
Date: Sat, 31 Jul 2021 15:46:18 +0200
Subject: [PATCH] CMakeLists.txt: CMake >= 3.11

* `INCLUDE_GUARD_NAME` in `GENERATE_EXPORT_HEADER()` was added in 3.11

Fixes: #216
---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc3a80c..2382802 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.11)
+# need 3.11 for INCLUDE_GUARD_NAME in GENERATE_EXPORT_HEADER
 
 project(
     BamTools
